admin_guide/metrics-sizing-guidelines.adoc

```diff
@@ -0,0 +1,164 @@
+= Openshift Metrics Sizing Guidelines
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Openshift Metrics Basics
+
+Monitoring openshift pods is important task from both, Openshift administrator and Openshift user point
+of view. Openshift administrators need a way to monitor pods of interest and see how they perform
+ and react if necessary to fix critical issue, at other side Openshift users also need simple and
+ comprehensive overview of pods where their application run.
+
+This guide will provide Openshift metrics guidlines which Openshift administrators and users can
+follow when trying to dimension openshift metrics for openshift installations
+
+== Openshift Metrics Installation
+
+Installation of Openshift metrics is described in https://docs.openshift.com/enterprise/3.2/install_config/cluster_metrics.html[Openshift Enterprise v3 documentation]
+For instruction how to configure Opensift Enterprise Metrics on functional Openshift cluster, please follow instruction
+in https://docs.openshift.com/enterprise/3.2/install_config/cluster_metrics.html[Openshift Enterprise documentation]
+
+However, it is very important to fulfill below points when starting Opensift metrics in order to avoid potential issues which can appear later during deployment
```

Comment:
It is important to also consider some additional factors that can help prevent issues during deployment.